simulator/generate_example_screenshots.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+OUTPUT_DIR=screenshots
```

Comment:
This should probably go in `../target/screenshots` (relative to the script dir), or at least be added to `.gitignore`.

Reply:
Writing files to a directory that isn't a subdirectory of the `simulator` directory, but of the parent directory, didn't "feel" right. But now that creating the screenshots doesn't require enabling a cargo feature anymore I was able to move the script into the root directory of the repository and use `target/screenshots` as an output directory.